ecosystem/platform/server/app/models/user.rb

```diff
@@ -11,7 +11,8 @@ class User < ApplicationRecord
          :omniauthable, omniauth_providers: %i[discord github google],
                         authentication_keys: [:username]
 
-  validates :username, uniqueness: { case_sensitive: false }, allow_nil: true
+  validates :username, uniqueness: { case_sensitive: false }, length: { maximum: 20 },
```

Comment:
can we validate minimum length of 3 as well? (discourse thing)

Reply:
Done.